crates/api_models/src/blocklist.rs

```diff
@@ -22,6 +32,11 @@ pub struct BlocklistResponse {
     pub created_at: time::PrimitiveDateTime,
 }
 
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "snake_case")]
+pub struct GenerateFingerprintResponsePayload {
+    pub card_fingerprint: String,
```

Comment:
Should we have this as a secret as well? or is it ok to log?

@vspecky @Chethan-rao  ?

Reply:
we'll have it as a Secret as we are following the same in locker

Reply:
I have a question @NishantJoshi00 ,  as we are already providing the fingerprint in response moreover storing them in intent/attempt table. What will be the problem associated with logging it ?